src/commonMethods/remoteWebInspector.js

```diff
@@ -4,9 +4,9 @@
 /*jslint esnext: true*/
 /*jslint node: true */
 'use strict'
-import constants from './constants'
 
 const WebSocket = require('ws')
+let host = '' //TODO - Update to get IP dynamically
```

Comment:
what is pending to make this host dynamic as well? the `this.$thunder.api.options.host` approach doesn't work here?

Reply:
Yes Michiel